src/opencl_sha256crypt_fmt_plug.c

```diff
@@ -44,6 +44,10 @@ static sha256_password *plain_sorted;   // sorted list (by plaintext len)
 static sha256_hash *calculated_hash;    // calculated hashes
 static sha256_hash *computed_hash;  // calculated hashes (from plain_sorted)
 
+static int
+*indices;            // relationship between sorted and unsorted plaintext list
+unsigned int bitmap_of_lens; // what plaintext sizes do we have?
```

Comment:
Add `static`, and I'd use an explicit width type like `uint32_t`. Also need a comment or ideally compile-time check that `PLAINTEXT_LENGTH` isn't larger than supported by this type (so not larger than e.g. 31 if we allocate bit 0 for length 0).